custodia/secrets.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, config, section):
 
     def _db_key(self, trail):
         if len(trail) < 2:
+            self.logger.error('Forbidden action. Trail: %s has less than two '
```

Comment:
Makes sense, I used the trail name because I saw it in the issue description, but makes totally sense provide a more comprehensive message for sysadmins.